### ISSUE TITLE
Quote blocks are not rendered correctly

### ISSUE THREAD

**Issue by yiufung** [kunalb/poet#10]
It seems that the face inside quote blocks is `org-meta-line` and hence not rendered correctly. 

![2019-01-16-160831_257x237](https://user-images.githubusercontent.com/1023375/51234946-315f3000-19a9-11e9-9125-65b13b6c0c53.png)
![2019-01-16-160848_456x532](https://user-images.githubusercontent.com/1023375/51234950-33c18a00-19a9-11e9-9f59-abc2d4b153c7.png)

Here's [a minimal config to reproduce.](https://gist.github.com/yiufung/bd4e7eecdf475f92c3f2a412a72ccd3c)

Version:
```
GNU Emacs 26.1 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.22.30) of 2018-07-05
Org mode version 9.2 (9.2-elpaplus @ /home/yiufung/.cyf-emacs/elpa/org-plus-contrib-20181230/)
```

**Comment by kunalb** [comment]
I took a look: the behaviour is apparently as intended by org, the default behaviour is to render anything inside verse/quote as pure org text, with appropriate highlighting for org syntax. If you'd like it to be consistent (like I do too) you'll have to set `org-fontify-quote-and-verse-blocks` ([example](https://github.com/kunalb/dot/blame/master/emacs/config.org#L120)).

Thanks for reporting! I cleaned up the dark mode quote rendering, and also fixed verse rendering thanks to this issue.
